src/main/resources/db/migration/V7__create_employee_role_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE Employee_Role (
+    role_id INT PRIMARY KEY NOT NULL,
+    employee_id INT PRIMARY KEY NOT NULL
```

Comment:
does it work with two Primary Keys like this?  I think this works like in PR 7: 
"PRIMARY KEY (project_id, technology_id)"